distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -44,7 +44,11 @@ LINUX="Linux"
 
 if [ "$OS" != "${OS%$LINUX*}" ]; then
    if [ -e /dev/urandom ]; then
-      EXTRA_JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
+      if [ -e "$EXTRA_JAVA_OPTS" ]; then
```

Comment:
I think you want to use the -z option here for a variable rather than a file.

Reply:
You are correct. Fixed.